core/src/main/java/hudson/cli/InstallPluginCommand.java

```diff
@@ -115,6 +116,7 @@ protected int run() throws Exception {
                 continue;
             } catch (MalformedURLException e) {
                 // not an URL
+                e.getCause();
```

Comment:
You need to handle a potential `URISyntaxException` here - calling `getCause()` is doing mostly nothing.